deployments/deploy-ssh.sh

```diff
@@ -312,14 +320,16 @@ spec:
     operator: Equal
     value: "false"
     effect: "NoSchedule"
-  affinity:
-    nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-        - matchExpressions:
-          - key: ${K8S_VENDOR_KEY}
-            operator: ${K8S_VENDOR_OPERATOR}
-            ${K8S_VENDOR_VALUE}
+#  affinity:
+#    nodeAffinity:
+#      preferredDuringSchedulingIgnoredDuringExecution:
+#      - weight: 100
+#        preference:
+#          matchExpressions:
+#          - key: "${K8S_VENDOR_KEY}"
+#            operator: "${K8S_VENDOR_OPERATOR}"
+#            values:
+#            - "${K8S_VENDOR_VALUE}"
```

Comment:
do we need this commented lines?

Reply:
No, we don't. I had experimenting with different affinity policies since I can't properly label k8s node to run torpedo.